CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 1.0.0-beta.1 (In Progress)
 ==========================
 
+## Changes
+- The integration testing app has been renamed/moved to `test/app` to avoid confusion as to the purpose of the app.
```

Comment:
Need to mention about what we moved, the `example` app. I don't think we need to mention the confusion part. Maybe something like
> The `example` app used for end to end testing has been moved to `test/app`.